about/index.md

```diff
@@ -1,15 +1,47 @@
 ---
-title: About
+title: About Forseti Security
 ---
-# About
+# {{ page.title }} 
 
-Forseti Security helps you secure your Google Cloud Platform (GCP) environment.
+A community-driven collection of open source tools to improve the security of
+your Google Cloud Platform environments.
 
-Features:
+[Get Started]({% link _docs/quickstarts/forseti-security/index.md %}) with
+Forseti Security.
 
-* **Visibility:** Build and store an inventory of your GCP resources.
-* **Detection:** Find policy violations across various resources in GCP.
-* **Enforcement:** Apply changes to enforce policy when needed.
+## [Inventory]({% link _docs/quickstarts/inventory/index.md %})
+
+Takes a snapshot of resources on a recurring cadence, so you always have a
+history of what was in your cloud.
+
+## [Policy Scanner]({% link _docs/quickstarts/scanner/index.md %})
+
```

Comment:
Do we want to make this a little more generic? We are not just scanning IAM policies; we also scan Cloud SQL authorized networks, bucket ACLs, etc.